### ISSUE TITLE
1.10.14 release is broken?

### ISSUE THREAD

**Issue by crgwbr** [ttag-org/ttag-cli#158]
After updating to ttag-cli 1.10.14, hitting this error:

```
ttag update path/to/foo.po bar/
node:internal/modules/cjs/loader:1228
  throw err;
  ^

Error: Cannot find module '../dist/src/index.js'
Require stack:
- […]/node_modules/ttag-cli/bin/ttag
    at Module._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Module._load (node:internal/modules/cjs/loader:1051:27)
    at Module.require (node:internal/modules/cjs/loader:1311:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> ([…]/node_modules/ttag-cli/bin/ttag:2:1)
    at Module._compile (node:internal/modules/cjs/loader:1469:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)
    at Module.load (node:internal/modules/cjs/loader:1288:32)
    at Module._load (node:internal/modules/cjs/loader:1104:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:173:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '[…]/node_modules/ttag-cli/bin/ttag'
  ]
}

Node.js v20.18.1
```

When inspecting the `node_modules` dir, the `node_modules/ttag-cli/dist/` indeed does not exist. Forget to compile before publishing?

**Comment by AlexMost** [comment]
Oh, I guess it was an accidental mistake. Fix is available i `ttag-cli@1.10.15`. Thanks for the report!

**Comment by AlexMost** [comment]
Thanks for the report. This was fixed in `1.10.18`
